### DISCUSSION TITLE
Why are both System.Text.Json and a 2 year old version of Newtonsoft.Json distributed with the WASDK package?

### DISCUSSION THREAD

**Discussion by kmgallahan** [microsoft/WindowsAppSDK#3490] [Q&A]
As per the title, why are both System.Text.Json and a 2 year old version of Newtonsoft.Json distributed with the WASDK package?

**Comment by DarranRowe** [comment]
Dependencies for the build tasks.
Some of these build tasks appear alongside Newtonsoft.Json in the Visual Studio install and others appear alongside System.Text.Json. Because of this, I would imagine that they used the "if it isn't broke then don't fix it" approach when they brought these build tasks into the Windows App SDK.
